drivers/hmis/app/views/hmis_client/services/_table.haml

```diff
@@ -0,0 +1,27 @@
+.table-responsive
+  %table.table
+    %thead
+      %tr
+        %th
+        %th Project
+        %th Entry Date
+        %th Service Date Provided
+        %th Service Name
+        %th
+    %tbody
+      - custom_services.each do |service|
+        %tr
+          %td
+            .ds{class: "ds-color-#{service.data_source_id}"}= ds_short_name_for(service.client.id)
+          %td= service.project.name
+          %td= service.enrollment.entry_date
+          %td= service.date_provided
+          %td= service.display_name
+          %td.text-right
+            - if @client.hmis_source_visible_by?(current_user)
+              = link_to 'HMIS', "https://#{service.enrollment.data_source.hmis}/client/#{service.enrollment.client.id}/enrollments/#{service.enrollment.id}/services", class: 'btn btn-xs btn-secondary btn-muted btn-hmis mb-2'
```

Comment:
I _think_ we want to use `service.data_source.hmis_url_for(service, user: current_user)`  The `hmis_url_for` method will need a case statement for the service, but that should be pretty straight forward with what you have.

Reply:
Ah wonderful. Thanks!